src/MiniJuvix/Syntax/MonoJuvix/Language/Extra.hs

```diff
@@ -0,0 +1,12 @@
+module MiniJuvix.Syntax.MonoJuvix.Language.Extra
+  ( module MiniJuvix.Syntax.MonoJuvix.Language.Extra,
+    module MiniJuvix.Syntax.MonoJuvix.Language,
+  )
+where
+
+import MiniJuvix.Syntax.MonoJuvix.Language
+
+foldApplication :: Expression -> [Expression] -> Expression
+foldApplication f args = case args of
```

Comment:
Use` \case`; `args` is not used

Reply:
This will be listed in the future by Hlint, I hope.

Reply:
I didn't use `\case` since by keeping the `args` name I thought it was more clear what the function does